### PR TITLE
Update DateTest.php

### PR DESCRIPTION
**Pull request by nimasdj**

### CODE REVIEW

File: tests/Query/Mysql/DateTest.php [hunk 1]
```diff
@@ -13,6 +13,15 @@ public function testDateDiff()
         $this->assertEquals($sql, $q->getSql());
     }
 
+    public function testFrom_Unixtime()
```
Comment (stevelacey) [inline tests/Query/Mysql/DateTest.php]: Can you drop the "_", `testFromUnixtime`

File: tests/Query/Mysql/DateTest.php [hunk 2]
```diff
@@ -13,6 +13,15 @@ public function testDateDiff()
...
+    {
+        $dql = "SELECT p FROM DoctrineExtensions\Tests\Entities\Date p WHERE FROM_UNIXTIME(CURRENT_TIME())";
+        $q = $this->entityManager->createQuery($dql);
+
+       $sql = "SELECT d0_.id AS id0, d0_.created AS created1 FROM Date d0_ WHERE FROM_UNIXTIME(CURRENT_TIME)";
```
Comment (stevelacey) [inline tests/Query/Mysql/DateTest.php]: I expect this will yield `CURRENT_TIME()`, that doesn't look like valid SQL

Comment (nimasdj) [inline tests/Query/Mysql/DateTest.php]: @stevelacey so what to do instead of current_time? how is valid sql?

Comment (stevelacey) [inline tests/Query/Mysql/DateTest.php]: I think if you just stick `()` on the end of `CURRENT_TIME` in on line 21 you should be good to go.

**Comment by nimasdj** [comment]
@stevelacey I did stick () and removed _ but it seems there is still another error, what else should I do?
it says it expect < = > 
where to add what?

**Pull request closed without merging**